### ISSUE TITLE
breaking symmetries

### ISSUE THREAD

**Issue by hklarner** [hklarner/RnaCancerClassifier#4]
Mir ist gerade aufgefallen, dass die Zuweisung von gate typen zu gate ids leider alle möglichen Permutationen zuläßt. Schau die zum Beispiel die optimalen toy modelle an, es gibt 4 Lösungen:

```
 g2 * !g1
!g2 *  g1

!g1 * !g3
!g3 * !g1
```

Das wollen wir natürlich nicht. Bin mir aber nicht sicher wie schwierig es ist, das durch Constraints rauszubekommen.

**Comment by katinkab** [comment]
Stimmt. Sind nur 2, 3 Zeile, denke ich. Aber die haben es in sich. Hab ich auch schon mal gemacht, kann ich aber auf die Schnelle nicht. Deshalb vielleicht nach morgen.
